### ISSUE TITLE
Create Email: invalid publishUp and publishDown

### ISSUE THREAD

**Issue by andremacola** [mautic/mautic#6199]
I tried all date formats and they do not work with the API, what I am doing wrong?

```
$emailCreateApi = $api->newApi('emails', $auth, $apiUrl);
$emailArgs = array(
  'emailType' => 'list',
  'subject' => 'Assunto do Email',
  'name' => 'Nome Interno do Email',
  'title' => 'Título do Email',
  'description'	=> 'Descrição do Email criado pela API',
  'isPublished'	=> 0,
  'publishUp' => "2018-06-12 05:17",
  'publishDown' => "2018-06-13 05:17",
  'lists' => array(3),
  'customHtml'	=> '<p>teste de html</p>'
);
$emailCreate = $emailCreateApi->create($emailArgs);
var_dump($emailCreate);
```

Already tried values like: 2018-06-16T06:00:00+00:00

```
code' => 400,
'message' => 'The response has unexpected status code (400).
"errors": [
    publishUp: This value is not valid....
]
```

**Comment by andremacola** [comment]
Fixed this issue by updating php 5.6 to 7.2 and installing mcrypt on php7.2

Tip: https://gist.github.com/arzzen/1209aa4a430bd95db3090a3399e6c35f

**Comment by ttencate** [comment]
The right format as of Mautic 4.1 seems to be `yyyy-MM-dd HH:mm`. But before, I used `2018-06-16T06:00:00+00:00` just like you, and it used to work.
